pennylane/devices/execution_config.py

```diff
@@ -67,6 +83,9 @@ class ExecutionConfig:
     derivative_order: int = 1
     """The derivative order to compute while evaluating a gradient"""
 
+    mcm_config: Optional[MCMConfig] = MCMConfig()
```

Comment:
```suggestion
    mcm_config: MCMConfig = MCMConfig()
```

Reply:
Aren't all `ExecutionConfig` arguments technically optional?

Reply:
Oh I see, just the ones that are `None` by default. Thanks for finding this, committing.